docs/conf.py

```diff
@@ -49,6 +49,11 @@
 author = 'The SunPy Community'
 copyright = '{}, {}'.format(datetime.datetime.now().year, author)
 
+
+# Register remote data option with doctest
```

Comment:
Does this need to go upstream to pytest-remote-data?

Reply:
If anywhere, to https://github.com/astropy/pytest-doctestplus - see discussion on https://github.com/matplotlib/matplotlib/issues/21668

Reply:
Right that one!